templates/instance/about.rs.html

```diff
@@ -13,7 +13,7 @@ <h1>@i18n!(ctx.1, "About {0}"; instance.name)</h1>
     <div class="banner">
         <section class="stats">
         <div>
-            <p>@Html(i18n!(ctx.1, "Home to <em>{0}</em> users"; n_users))</p>
+            <p>@Html(i18n!(ctx.1, "Home to <em>{0}</em> people"; n_users))</p>
```

Comment:
👍

would be easier to remember if `n_users` was `n_people`

Reply:
It's honestly not a big problem. I think from a programmer's perspective, having the declaration as `users` is easier to understand in code maybe?

Reply:
true, true…